native/src/rendering.cpp

```diff
@@ -1027,6 +1028,22 @@ double polygonArea(MapDataObject* obj, float mult) {
 	return abs(area) * mult * mult * .5;
 }
 
+/*int getSquareSegmentLength(MapDataObject* obj) {
```

Comment:
no commented code, - please

Reply:
It was specially left for review. See comment below.